plugins/inputs/cloudwatch/README.md

```diff
@@ -4,60 +4,50 @@ This plugin will pull Metric Statistics from Amazon CloudWatch.
 
-1. Assumed credentials via STS if `role_arn` attribute is specified (source credentials are evaluated from subsequent rules)
+1. Assumed credentials via STS if `role_arn` attribute is specified (source
+   credentials are evaluated from subsequent rules)
 2. Explicit credentials from `access_key`, `secret_key`, and `token` attributes
 3. Shared profile from `profile` attribute
-4. [Environment Variables](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#environment-variables)
-5. [Shared Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#shared-credentials-file)
-6. [EC2 Instance Profile](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html)
+4. [Environment
+   Variables](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#environment-variables)
+5. [Shared
+   Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#shared-credentials-file)
+6. [EC2 Instance
+   Profile](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html)
 
 ## Configuration
 
 ```toml @sample.conf
 # Pull Metric Statistics from Amazon CloudWatch
-[[inputs.cloudwatch]]
-  ## Amazon Region
-  region = "us-east-1"
-
-  ## Amazon Credentials
-  ## Credentials are loaded in the following order
-  ## 1) Web identity provider credentials via STS if role_arn and web_identity_token_file are specified
-  ## 2) Assumed credentials via STS if role_arn is specified
-  ## 3) explicit credentials from 'access_key' and 'secret_key'
-  ## 4) shared profile from 'profile'
-  ## 5) environment variables
-  ## 6) shared credentials file
-  ## 7) EC2 Instance Profile
-  # access_key = ""
-  # secret_key = ""
-  # token = ""
-  # role_arn = ""
-  # web_identity_token_file = ""
-  # role_session_name = ""
-  # profile = ""
-  # shared_credential_file = ""
-
-  ## Endpoint to make request against, the correct endpoint is automatically
-  ## determined and this option should only be set if you wish to override the
-  ## default.
-  ##   ex: endpoint_url = "http://localhost:8000"
-  # endpoint_url = ""
-
-  ## Set http_proxy (telegraf uses the system wide proxy settings if it's is not set)
-  # http_proxy_url = "http://localhost:8888"
+[[inputs.cloudwatch]] ## Amazon Region region = "us-east-1"
+
+  ## Amazon Credentials ## Credentials are loaded in the following order ## 1)
+  Web identity provider credentials via STS if role_arn and
+  web_identity_token_file are specified ## 2) Assumed credentials via STS if
+  role_arn is specified ## 3) explicit credentials from 'access_key' and
+  'secret_key' ## 4) shared profile from 'profile' ## 5) environment variables
+  ## 6) shared credentials file ## 7) EC2 Instance Profile # access_key = "" #
+  secret_key = "" # token = "" # role_arn = "" # web_identity_token_file = "" #
+  role_session_name = "" # profile = "" # shared_credential_file = ""
```

Comment:
Was this auto-generated fixed? as the numbers should probably be kept on different lines and the `##` are already there